app/views/job_templates/cloud_connector.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Configure Cloud Connector
+snippet: false
+template_inputs:
+- name: satellite_cloud_connector_user
```

Comment:
The input names were different. Either they need to stay the same or the rename needs to be done here https://github.com/theforeman/foreman_rh_cloud/blob/master/app/services/foreman_rh_cloud/cloud_connector.rb#L19 as well

Reply:
Great catch, fixed!